docs/node/production-guide.mdx

```diff
@@ -0,0 +1,392 @@
+
+#### Starting as a Sentry Node
+
+Now we will add a third note to the network as a sentry node. Since this node is not a genesis validator, its public key is not included in the genesis file, which is now immutable. The `kwil-admin setup init` command can be used to simultaneously initialize a new node's root directory and generate a new private key.
+
+:::important
+When adding a node to the network, start it as a non-validator / sentry node.
+Always completely synchronize the node to the network before attempting to promote it to a validator.
+:::
+
+Like before we will specify the location of the network's `genesis.json` file, the host's external IP address, and one or more persistent peers. However, we will *not* specify a private key path so that a new key is generated:
+
+```sh title="node2"
+[node2]:~$ kwil-admin setup init --genesis prod-chain/genesis.json \
+  --chain.moniker "node2" \
+  --chain.p2p.external-address "2.3.4.5:26656" --chain.p2p.persistent-peers \
+    "2164057e1d8248f42aadb8ef00afcc123180571d@98.76.54.32:26656" \
+  --root-dir /data/kwil-node2
+Initialized node in /data/kwil-node2
+```
+
+We can see the ID of `node2` before starting it:
+
+```sh title="node2"
+[node2]:~$ kwil-admin key info --key-file /data/kwil-node2/private_key | grep "Node ID"
+Node ID: bd56ce5a93c47b532be71531ff6ed469b341e61f
+```
+
+#### Verifying Successful Node Synchronization
+
+Before adding the node as a validator, ensure that `node2` has completely synchronized with the network. You can do this with either the `kwil-admin` tool, or with the public user service using `kwil-cli`.
+
+Using `kwil-admin node status`, insect the `.sync.best_block_height` and `.sync.syncing` values:
+
+```sh title="node2"
+[node2]:~$ kwil-admin node status
+{
+  "node": {
+    "chain_id": "prod-chain",
+    "name": "node2",
+    "node_id": "bd56ce5a93c47b532be71531ff6ed469b341e61f",
+    "proto_ver": 8,
+    "app_ver": 1,
+    "block_ver": 11,
+    "listen_addr": "tcp://127.0.0.2:27656",
+    "rpc_addr": "tcp://127.0.0.2:26657"
+  },
+  "sync": {
+    "app_hash": "0e0674b0b88abc26cc60da6e64883dfef802f3431f19296b88c8bc1db8f4f4ae",
+    "best_block_hash": "1d5a4ef85927dce3c90168e89fdfb32219c5c95bb5ed3332eb69e20bbe7346da",
+    "best_block_height": 109,
+    "best_block_time": "2024-10-03T18:01:38.585-05:00",
+    "syncing": false
+  },
+  "validator": {
+    ...
+  },
+  "app": null
+}
+```
+
+Ensure that the node is no longer syncing (catching up) with the network and that the best block height is within a couple blocks of the network's known height.
+
+Use `kwil-cli utils chain-info` to query the height of any node with its RPC service exposed:
+
+```sh
+$ kwil-cli utils chain-info --provider "http://98.76.54.32:8484"
+Chain ID: prod-chain
+Height: 109
+Hash: 1d5a4ef85927dce3c90168e89fdfb32219c5c95bb5ed3332eb69e20bbe7346da
+```
+
+Compare the output of that command with the result from `node2` to ensure the new node is keeping up.
```

Comment:
Nit:

```suggestion
Compare the output of that command with the result from `node2` to ensure the new node has completed syncing.
```

Reply:
Will rephrase this, but "completed syncing" doesn't send the right message IMO.  There's a initial block download e.g. catch-up on startup, but a nodes is continually syncing/staying in sync with the network and that's what we want to observe

Reply:
Maybe: "is in sync with the rest of the network" ?